act-rules-format.bs

```diff
@@ -52,18 +52,21 @@ The separation between atomic rules and composite rules creates a division of re
     <li>Header indicated by using the `headers` attribute</li>
     <li>Header indicated by using ARIA labels</li>
   </ul>
-  <p>If any one of these rules passes, the table cell can pass the composite rule.</p>
+  <p>If any one of these rules passes, the test result of the composite rule for the table cell is "passed".</p>
 </div>
 
 
 ACT Rule Structure {#act-rule-structure}
 ===============================
 
-An ACT Rule MUST consist of the following items
+ACT Rules MUST be written in an format that conforms to the Web Content Accessibility Guidelines [[WCAG]] or a comperable accessibility guidelines recommendation. ACT test cases are allowed to contain inaccessible content. If any test case contains accessibility issues listed in [WCAG 2.1 Section 5.2.5 Non-Interference](https://www.w3.org/TR/WCAG21/#cc5), users MUST be warned of this in advance. Using an accessibility format supports people with disabilities. It also makes internationalisation of ACT rules easier.
```

Comment:
I am not sure how to understand this section. 
When you say "written in a format..." I am thinking it should be the ACT Rules Format they conform to.
Is the intent of this section that the presentation of the ACT rule should be accessible itself?